app/models/user.rb

```diff
@@ -637,6 +637,10 @@ def banned_at_date?(date)
       end
     end
 
+    if can_manage_delegate_probation?
+      groups += UserGroup.delegate_probation.map(&:id)
```

Comment:
I think `UserGroup.delegate_probation.ids` should work (if `delegate_probation` is a Rails scope and not a custom method returning an array)

Reply:
Thanks, didn't knew about this. :-)